src/helpers/uriHandlerController.ts

```diff
@@ -0,0 +1,47 @@
+import {startDebugging} from '@/commands';
+import {Constants} from '@/Constants';
+import {Disposable, Uri, UriHandler, window} from 'vscode';
+
+type TDebugInformation = {
+  txHash: string;
+  workingDirectory: string;
+  providerUrl: string;
+};
+
+enum Commands {
+  debug = 'debug',
+}
+
+export default class UriHandlerController implements UriHandler {
+  private disposables: Disposable[] = [];
```

Comment:
What are the `disposables` used for? AFAIU, the only disposable here is the value returned by `registerUriHandler`.